tools/glyph-studio/py/typography_runs.py

```
@@ -0,0 +1,796 @@
+        for pr in per_receipt:
+            write_run_map(args.out_dir, slug, pr)
+        viz_paths = [
+            render_overlay(args.out_dir, slug, pr, client, name)
+            for pr in per_receipt[: args.viz]
+        ]
+        ex_path, _, _ = render_exemplars(args.out_dir, slug, exemplars, atlas)
+
+        # aggregates
+        scores = [
+            l["attribution"]
+            for _, l, _ in all_lines
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)